python/ray/ml/train/integrations/sklearn/sklearn_trainer.py

```diff
@@ -434,3 +436,24 @@ def training_loop(self) -> None:
             "fit_time": fit_time,
         }
         tune.report(**results)
+
+    @staticmethod
+    def load_checkpoint(
+        checkpoint: Checkpoint,
+    ) -> Tuple[BaseEstimator, Optional[Preprocessor]]:
+        """Load a Checkpoint from ``SklearnTrainer``.
+
+        Return the estimator and AIR preprocessor contained within.
```

Comment:
we switch wording between "estimator" and "model" in doc strings. I think we should settle for one .

Reply:
"Estimator" is sklearn parlance, it should be used specifically for that